src/assets/styleJs/events/events.ts

```diff
@@ -0,0 +1,53 @@
+import {makeStyles} from '@mui/styles'
+import {color, font, margin} from '../../style/globalVariables'
+
+const useEventsStyle = makeStyles({
+  ongoingEventsDivider: {
+    margin: `${margin.marginTopBottom20} ${margin.marginLeftRight0}`,
+    backgroundColor: color.lightGrey,
+  },
+  eventContent: {
+    height: ' calc(100vh - 132px)',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexDirection: 'column',
+  },
+
```

Comment:
Spacing is not correct, Please check your local eslint and prettier configs
@yavetisyan @gor017